### ISSUE TITLE
Should we be telling people to use temporary branches and git amend?

### ISSUE THREAD

**Issue by odscjames** [OpenDataServices/developer-docs#61]
https://github.com/OpenDataServices/developer-docs/blob/master/tools/git.md#temporary-branches

> In general having temporary commits on temporary branches is better than using git stash. ... they can also be pushed to the server. ....  If you have a temporary commit you can update it using git commit --amend -a --date="date``

If you push to a remote, git amend and then try and push again that's not great? Maybe take advice out about  commit --amend?

(Also maybe tell people to delete temporary branches from GitHub when done?)

**Comment by BibianaC** [comment]
Temporary branches with temporary commits, does this mean to always work in branches instead of master? 
+1 on deleting them once done.

**Comment by odscjames** [comment]
Yes, the first section of https://github.com/OpenDataServices/developer-docs/blob/master/tools/git.md specifies feature branches. (Actually it specifies "most", we should maybe clarify that later!)

**Comment by BibianaC** [comment]
It would be good to clarify this 'must'. Specify what is allowed _not_ to be in a branch.
